pkg/agent/openflow/pipeline.go

```diff
@@ -2702,6 +2702,16 @@ func (f *featureMulticast) localMulticastForwardFlows(multicastIP net.IP, groupI
 	}
 }
 
+// externalFlexibleIPAMMulticastReceiverFlow generate the flow similar to the externalMulticastReceiverFlow function,
```

Comment:
generates

a flow

Reply:
the externalMulticastReceiverFlow function -> externalMulticastReceiverFlow

Reply:
updated